README.md

```diff
@@ -73,3 +79,67 @@ You can run node scripts described in package.json using Yarn by setting `$(Yarn
     <YarnBuildCommand>build</YarnBuildCommand>
 </PropertyGroup>
 ```
+
+### Jasmine installation
+
+If you want to use a jasmine testing framework, you may specify jasmine version by setting `$(JasmineVersion)`, otherwise jasmine will not be installed.
+
+```xml
+<PropertyGroup>
+    <JasmineVersion>latest</JasmineVersion>
+</PropertyGroup>
+```
+
+Jasmine command stored to `$(JasmineExecutable)` property. This target (`InstallJasmine`) depends on `InstallNodeJs`.
+
+### Generate jasmine configuration and executable shell file
+
+If you installed jasmine, in the output directory of your project will be generate configuration file (`jasmine.json`) and shell file which execute jasmine with custom reporter([`SimpleReporter.js`](./src/Redistributable/reporters/SimpleReporter.js)) and configuration file. This target (`GenerateJasmineConfigAndStartup`) depends on `InstallJasmine`.
+
+Example of jasmine.json configuration:
+
+```json
+{
+	"spec_dir": ".",
+	"spec_files": [
+			"/path/to/out/dir/fail.test.js",
+			"/path/to/out/dir/simple.test.js"
+		],
+	"stopSpecOnExpectationFailure": "false",
+	"random": "false"
+}
+```
+
+For more detail information about configuration file [click this](https://jasmine.github.io/pages/docs_home.html).
+
+# Adeptik NodeJs UnitTesting TestAdapter 
+
+## About package
+
+Infrastructure to run unit test using Jasmine javascript testing framework. Depends on Adeptik.NodeJs.Redistributable to run tests without NodeJS installed globally.
+
+Supports Windows, Linux & MacOS.
+
+![Build Status](https://tfs.adeptik.com/Adeptik/_apis/public/build/definitions/5f6da651-409b-4516-b0c6-16518d60e6e9/137/badge)
+[![Nuget Package](https://img.shields.io/nuget/vpre/Adeptik.NodeJs.UnitTesting.TestAdapter)](https://www.nuget.org/packages/Adeptik.NodeJs.UnitTesting.TestAdapter)
+
+## Usage
+
+### Package installation
+
+Install Adeptik.NodeJs.UnitTesting.TestAdapter package from Nuget
+
+    Install-Package Adeptik.NodeJs.UnitTesting.TestAdapter -Version 1.1.0
+
+or add in your .csproj:
+
+    <PackageReference Include="Adeptik.NodeJs.UnitTesting.TestAdapter" Version="1.1.0" />
+
+### How it's work?
+
+This package implements a standard mechanism for finding and executing test cases by creating your own test adapter for Jasmine testing.
+
+### Require:
+
+* At build time, all jasmine test cases should be in the assembly output directory
+* All test case output files must adhere to the following name format `nameOfYourTestFile.test.js`
```

Comment:
Note, that all jasmine tests runs for test discovering & even when you try run exactly specified tests in test explorer.